kie-server-parent/kie-server-services/kie-server-services-common/pom.xml

```diff
@@ -15,8 +15,46 @@
 
   <properties>
     <java.module.name>org.kie.server.services.common</java.module.name>
+    
+    <io.fabric8.kubernetes-client.version>4.0-SNAPSHOT</io.fabric8.kubernetes-client.version>
```

Comment:
@rhtevan Can you please use any final version, otherwise the artifacts aren't resolved in the maven build.
Please use artifacts available in https://repository.jboss.org/nexus/content/groups/public/

Reply:
Committed the change as per suggested